video2dataset/data_reader.py

```diff
@@ -57,33 +57,43 @@ def get_yt_meta(url, yt_metadata_args: dict) -> dict:
     yt_metadata_args is a dict of follwing format:
     yt_metadata_args = {
         'writesubtitles': True,
+        'writealllangs': True,
         'subtitleslangs': ['en'],
         'writeautomaticsub': True,
         'get_info': True
     }
 
     writesubtitles:    Whether to write subtitles
+    writealllangs:     Whether to write subtitles for each provided language or just the first present
```

Comment:
hmm maybe instead of this we should just make a single parameter for this. Maybe something like 'subtitleslangs' and it can either be a list of languages or 'first' or 'all'

Reply:
that seems reasonable, will update when I get a chance